icons/Makefile.am

```diff
@@ -4,30 +4,16 @@ public_icons_themes = \
 	hicolor	\
 	$(NULL)
 
-private_icons = \
-		hicolor_status_16x16_abrt.png	\
-		hicolor_status_22x22_abrt.png	\
-		hicolor_status_24x24_abrt.png	\
-		hicolor_status_32x32_abrt.png	\
-		hicolor_status_48x48_abrt.png	\
-		hicolor_status_256x256_abrt.png
-		$(NULL)
-
-public_icons = hicolor_apps_16x16_abrt.png	\
-		hicolor_apps_22x22_abrt.png	\
-		hicolor_apps_24x24_abrt.png	\
-		hicolor_apps_32x32_abrt.png	\
-		hicolor_apps_48x48_abrt.png	\
-		hicolor_status_16x16_abrt.png	\
-		hicolor_status_22x22_abrt.png	\
-		hicolor_status_24x24_abrt.png	\
-		hicolor_status_32x32_abrt.png	\
-		hicolor_status_48x48_abrt.png	\
-		hicolor_status_256x256_abrt.png
-		$(NULL)
+public_icons = \
+	hicolor_apps_scalable_abrt.svg \
+	hicolor_apps_symbolic_abrt-symbolic.svg \
+	$(NULL)
 
 
-EXTRA_DIST = $(private_icons) $(public_icons)
+EXTRA_DIST = \
+	$(public_icons) \
+	hicolor_apps_48x48_abrt.png \
```

Comment:
Shouldn't this be removed?

Reply:
Nuh-uh, it’s used in the docs as the project logo. Maybe the larger SVG could be used, but I don’t know how it looks, because my impression was that it wouldn’t get scaled.

Reply:
Ohh, you've **replaced** the `hicolor_apps_48x48_abrt.png`. In the GitHub's UI it looks as the file was removed. That confused me.

Reply:
Yeah, it’s not great.